docker-compose-develop.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ../quiz-api:/app
+    command: sh -c "python manage.py runserver 0.0.0.0:8000"
+    environment:
+      - DEBUG = True
+      - SECRET_KEY = 'django-insecure-q9*+psd#d+exp$x5ec(t-1kpo8%aj_tm#hkzg=s1ath*(^mgge'
```

Comment:
eisob variables environment file theke nile better, github e tokens and keys expose kora not good.